public/components/visualization/text2viz.tsx

```diff
@@ -421,13 +421,13 @@ export const Text2Viz = () => {
   const getInputSection = () => {
     return (
       <>
-        <EuiFlexItem grow={2} style={{ width: 0 }}>
+        <EuiFlexItem grow={1} style={{ width: 0 }}>
```

Comment:
Nit: I remember we set source selector to use more free space than question input to avoid index pattern names being overlapped. Are there any requirements that changed for updating the selector width here?